contracts/test/Rollup.t.sol

```diff
@@ -224,11 +230,212 @@ contract RollupTest is RollupBaseSetup {
         }
     }
 
+    ////////////////
+    // Admin
+    ///////////////
+
+    function testFuzz_addValidatorsToWhitelist_succeeds() external {
+        // Initialize rollup with an empty validator whitelist
+        _initializeRollup(
+            0, // confirmationPeriod
+            0, // challengePeriod
+            0, // minimumAssertionPeriod
+            1, // baseStakeAmount,
+            0, // initialAssertionID
+            0, // initialInboxSize
+            new address[](0) // validator whitelist
+        );
+
+        address[] memory validators = new address[](1);
+        validators[0] = alice;
+
+        // Adding a validator to whitelist as non-admin should fail
+        vm.expectRevert(
```

Comment:
We can simplify this test with a selector in the next OpenZeppelin version which defines custom error.